sagemaker_neo_compilation_jobs/deploy_mxnet_model_on_Inf1_instance/mxnet_distributed_mnist_neo_inf1.ipynb

```diff
@@ -0,0 +1,277 @@
+    "\n",
+    "### Inf 1 instances \n",
+    "Inf1 instances are built from the ground up to support machine learning inference applications and feature up to 16 AWS Inferentia chips, high-performance machine learning inference chips designed and built by AWS. The Inferentia chips are coupled with the latest custom 2nd generation Intel® Xeon® Scalable processors and up to 100 Gbps networking to enable high throughput inference. With 1 to 16 AWS Inferentia chips per instance, Inf1 instances can scale in performance to up to 2000 Tera Operations per Second (TOPS) and deliver extremely low latency for real-time inference applications. The large on-chip memory on AWS Inferentia chips used in Inf1 instances allows caching of machine learning models directly on the chip. This eliminates the need to access outside memory resources during inference, enabling low latency without impacting bandwidth. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ruled-billion",
+   "metadata": {},
+   "source": [
+    "### Set up the environment\n",
+    "We need to first upgrade the SageMaker SDK for Python to v2.33.0 or greater & restart the kernel"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "brilliant-attempt",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!~/anaconda3/envs/mxnet_p36/bin/pip install --upgrade sagemaker"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "automotive-cream",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import sagemaker\n",
+    "from sagemaker import get_execution_role\n",
+    "from sagemaker.session import Session\n",
+    "\n",
+    "sagemaker_session = sagemaker.Session()\n",
+    "# S3 bucket for saving code and model artifacts.\n",
+    "# Feel free to specify a different bucket here if you wish.\n",
+    "bucket = Session().default_bucket()\n",
+    "\n",
+    "# Location to save your custom code in tar.gz format.\n",
+    "custom_code_upload_location = 's3://{}/customcode/mxnet'.format(bucket)\n",
+    "\n",
+    "# Location where results of model training are saved.\n",
+    "model_artifacts_location = 's3://{}/artifacts'.format(bucket)\n",
+    "\n",
+    "# IAM execution role that gives SageMaker access to resources in your AWS account.\n",
+    "# We can use the SageMaker Python SDK to get the role from our notebook environment. \n",
+    "role = get_execution_role()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "patient-benjamin",
+   "metadata": {},
+   "source": [
+    "# Construct a script for distributed training \n",
+    "The mnist.py script provides all the code we need for training and hosting a SageMaker model."
```

Comment:
Add a quick sentence why you are creating a script for distributed training. (as opposed to non-distributed)

Reply:
Thanks for the comment, it is actually not needed. I changed to default(non-distributed) training.

Reply:
Awesome. Thanks!

Reply:
Thank you for the review! Could you help merge this PR?